optuna/trial/_trial.py

```diff
@@ -696,7 +702,7 @@ def system_attrs(self) -> Dict[str, Any]:
             A dictionary containing all system attributes.
         """
 
-        return copy.deepcopy(self.storage.get_trial_system_attrs(self._trial_id))
+        return self._cached_frozen_trial.system_attrs
```

Comment:
Could you returns the `self._cached_frozen_trial.user_attrs` in `user_attrs` property method?

In terms of `system_attrs`, please keep the existing logic since it will be deprecated.

Reply:
I revised the code.
https://github.com/eukaryo/optuna/commit/08b951cbb4586af496db48bbf07b5bf4c5233b8c